cc1101/__init__.py

```diff
@@ -177,6 +181,7 @@ def __init__(
             >>>     # lock removed
         """
         self._spi = spidev.SpiDev()
+        self.max_speed_hz = int(max_speed_hz)
```

Comment:
The attribute should probably be protected/private (prefix `_`), cause we currently to not support changing its value on the fly (instead requires re-opening context by calling `__enter__`)
```suggestion
        self._spi_max_speed_hz = spi_max_speed_hz
```

Reply:
I have no preference on this one. If you think it's more clear this way I'll go for that. I'll do it